apps/sms/index.html

```diff
@@ -84,6 +84,9 @@
     <script defer src="views/shared/js/settings.js"></script>
     <script defer src="services/js/message_manager.js"></script>
     <script defer src="views/shared/js/app.js"></script>
+    <script defer src="services/activity/threads.js"></script>
```

Comment:
Yep, I remember you wrote that you were discussing this with Wilson. Did you want to have "threads.js" sources somewhere in shared (so that everyone can use it and update new versions only there) and we use browserify as build step? Or you want to have our own browserified version that we should update by ourselves?